doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -12,6 +12,7 @@ The Uno Platform Hot Reload feature provides a way to modify the XAML and C# of
 - XAML and [C# Markup](xref:Uno.Extensions.Markup.Overview) Hot Reload for **iOS, Catalyst, Android, WebAssembly, and Skia (X11, Windows, macOS and FrameBuffer)**
```

Comment:
@dr1rrb / @rajamatt / @jeromelaban 
Outside the addition of the documentation for the Hot Reload Indicator, do we have additional updates needed for this file for Hot Reload in general, please?

- https://platform.uno/docs/articles/features/working-with-xaml-hot-reload.html
- https://platform.uno/docs/articles/uno-development/uno-internals-hotreload.html